src/js/vic-v2/components/IntroductionPage.jsx

```diff
@@ -5,10 +5,36 @@ import { connect } from 'react-redux';
 import { focusElement } from '../../common/utils/helpers';
 import FormTitle from '../../common/schemaform/components/FormTitle';
 import SaveInProgressIntro, { introActions, introSelector } from '../../common/schemaform/save-in-progress/SaveInProgressIntro';
+import { hasSavedForm, scrollToElement, handleScrollEnd } from '../helpers';
 
 class IntroductionPage extends React.Component {
   componentDidMount() {
+    this.timer = null;
     focusElement('.va-nav-breadcrumbs-list');
+    window.addEventListener('scroll', this.onScroll);
+  }
+
+  componentDidUpdate(prevProps) {
+    const { savedForms } = this.props.saveInProgress.user.profile;
+    const { formId } = this.props.saveInProgress;
+    const notPreviouslyLoggedIn = !prevProps.saveInProgress.user.login.currentlyLoggedIn;
+    if (notPreviouslyLoggedIn && hasSavedForm(savedForms, formId)) {
+      this.scrollingToTop = true;
+      scrollToElement('.va-nav-breadcrumbs-list');
+    }
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('scroll', this.onScroll);
+  }
+
+  onScroll = () => {
```

Comment:
It looks like this is to focus after scrolling is finished? I think you can get away with scrolling and then focusing in `componentDidUpdate`. It shouldn't cause any problems to have the focus set before scrolling is finished.

Reply:
Thanks @jbalboni, great idea.